mesonbuild/dependencies/__init__.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2017 The Meson development team
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ._base import *
+from ._dev import GMockDependency, GTestDependency, ValgrindDependency
+from ._misc import BoostDependency, Python3Dependency, ThreadDependency
+from ._platform import AppleFrameworks
+from ._ui import GLDependency, GnuStepDependency, Qt4Dependency, Qt5Dependency, SDL2Dependency, WxDependency
+
+
+packages.update({
```

Comment:
__[flake8]__
* [F405] 'packages' may be undefined, or defined from star imports: ._base

<!-- Comment by [SideCI](https://sideci.com) -->